expan/core/experiment.py

```diff
@@ -195,12 +198,13 @@ def filter(self, kpis, percentile=99.0, threshold_type='upper'):
 
         self.data = self.data[flags == False]
 
-    def sga(self, feature_name_to_bins):
+    def sga(self, feature_name_to_bins, multi_test_correction=False):
         """
         Perform subgroup analysis.
         Args:
             feature_name_to_bins (dict): a dict of feature name (key) to list of Bin objects (value). 
                                       This dict defines how and on which column to perform the subgroup split.
+            multi_test_correction (boolean): True or False whether the correction for multiple testing is needed.
```

Comment:
how about "flag of whether the correction for multiple testing is needed"?

Reply:
sure, will make everywhere the same.